Lib/test/test_rlcompleter.py

```diff
@@ -91,7 +91,7 @@ def bar(self):
         f = Foo()
         completer = rlcompleter.Completer(dict(f=f))
         self.assertEqual(completer.complete('f.b', 0), 'f.bar')
-        self.assertEqual(f.calls, 1)
+        self.assertLessEqual(f.calls, 1)
```

Comment:
This weakens the test significantly. Could we rather split the two cases into separate tests?